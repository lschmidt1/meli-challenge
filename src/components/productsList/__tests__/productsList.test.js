import { shallow, mount, render } from "enzyme";
import React from "react";
import { Provider } from "react-redux";
import store from "../../../redux/store";
import ProductsList from "../productsList";

describe("<ProductsList />", () => {
  let component;
  beforeEach(() => {
    component = mount(
      <Provider store={store}>
        <ProductsList />
      </Provider>
    );
  });
  describe("should render correctly", () => {
    test("by matching snapshot", () => {
      expect(component).toMatchSnapshot();
    });
  });

  test("if there are no products loaded, list should be null", () => {
    expect(component.find("ul").exists()).toBe(false);
  });
});
