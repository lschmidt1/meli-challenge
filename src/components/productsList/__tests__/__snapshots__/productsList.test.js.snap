// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductsList /> should render correctly by matching snapshot 1`] = `ReactWrapper {}`;
